src/main/plugin/iso19139.ca.HNAP/present/metadata.xsl

```diff
@@ -15,14 +15,15 @@
                 xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#"
                 xmlns:skos="http://www.w3.org/2004/02/skos/core#"
                 xmlns:svrl="http://purl.oclc.org/dsdl/svrl"
-                exclude-result-prefixes="gmx xsi gmd gco gml gts srv xlink exslt geonet rdf ns2 rdfs skos svrl">
+                xmlns:saxon="http://saxon.sf.net/"
+                extension-element-prefixes="saxon"
+                exclude-result-prefixes="gmx xsi gmd gco gml gts srv xlink exslt geonet rdf ns2 rdfs skos svrl saxon">
 
   <xsl:import href="metadata-brief.xsl"/>
   <xsl:import href="metadata-fop.xsl"/>
   <xsl:import href="metadata-utils.xsl"/>
-  <xsl:include href="iso19139.ec.HNAP-readonly.xsl"/>
 
-  <!-- main template - the way into processing iso19139.ca.HNAP -->
+  <!-- main template - the way into processing iso19139.nap -->
```

Comment:
Is there a reason the comment was changed? It seems like the older one was correct.